.github/workflows/release-series-images.yml

```diff
@@ -20,62 +20,47 @@ jobs:
       id: mkdatetag
       run: echo "::set-output name=dtag::$(date +%Y%m%d-%H%M)"
 
-  base-image-build:
+  build-images:
     name: compute-entrypoint:${{ matrix.osg_series }}-${{ matrix.repo }} image build
```

Comment:
```suggestion
```

This name is no longer correct: we should have 6 tests (3 repos * 2 images). GHA will give us a pretty sensible default name (it shows up in the list of jobs on the left here https://github.com/opensciencegrid/docker-compute-entrypoint/pull/79/checks)